test/test_analysis.py

```diff
@@ -189,5 +189,71 @@ async def test_get_sample_cram_path_map_for_seqr(self):
             ],
         )
 
-        id_map = await self.al.get_sample_cram_path_map_for_seqr(self.project_id, ['blood'], [part[0].id])
+        id_map = await self.al.get_sample_cram_path_map_for_seqr(
```

Comment:
This is still on me for not checking why my vscode settings are out of whack with CI